vivisect/__init__.py

```diff
@@ -212,11 +212,12 @@ def getFref(self, va, idx):
         """
         return self.frefs.get((va,idx))
 
-    def getEmulator(self, logwrite=False, logread=False):
+    def getEmulator(self, logwrite=False, logread=False, taintbyte='A'):
```

Comment:
on what class would you like to see the setter? the `taintbyte` is a property of an emulator instance, so my first guess is that it would go on `Emulator`; however, this is the factory for emulator instances and it doesn't feel so ergonomic:


strategy one:
```
# taintbyte configuration is global for all emulators
vw.setTaintByte('A')
emu = vw.getEmulator()
...
```

strategy two:
```
emu = vw.getEmulator()
# taintbyte is local to each emulator
emu.setTaintByte('A')
...
```

strategy three (current PR):
```
# shortcut for above
emu = vw.getEmulator(taintbyte='A')
...
```

I'd suggest that (2) is most flexible, and (3) the easiest to use. What would you like to see?

Reply:
both.  make both possible, and the getEmulator() calls emu.setTaintByte(taintbyte)